packages/ui/src/components/no-data.tsx

```diff
@@ -60,6 +63,9 @@ export const NoData: FC<NoDataProps> = ({
 }) => {
   const { NavLink } = useRouterContext()
 
+  const PrimaryWrapper = primaryButton?.isDialogTrigger ? Dialog.Trigger : Fragment
```

Comment:
Can we name it more explicitly? Like `PrimaryDialogTrigger` and `SecondaryDialogTrigger`?